src/components/publicApi/LoginUiProvider.tsx

```diff
@@ -6,12 +6,21 @@ import { Airship } from '../services/AirshipInstance'
 import { changeTheme, getTheme, ThemeProvider } from '../services/ThemeContext'
 
 /**
- * We use this context to determine if `LoginUiProvider` is mounted.
+ * We use this context to determine if `LoginUiProvider` is mounted, and whether
+ * we are on mobile or desktop.
  */
-const loginUiContext = React.createContext(false)
+export const loginUiContext = React.createContext({
+  hasProvider: false,
+  isDesktop: false
+})
+
+export function useLoginUi(): { hasProvider: boolean; isDesktop: boolean } {
+  return React.useContext(loginUiContext)
+}
```

Comment:
This `useLoginUi` wrapper isn't used anywhere, so let's just drop it. It doesn't improve usability much.